benchexec/tools/korn.py

```diff
@@ -34,8 +37,21 @@ def version(self, executable):
     def name(self):
         return "Korn"
 
-    def project_url(self):
-        return "https://github.com/gernst/korn"
+    def cmdline(self, executable, options, task, rlimits):
+        cmd = [executable]
+        cmd = cmd + options
+
+        data_model = task.options.get("data_model")
+
+        if data_model == "ILP32":
+            cmd = cmd + ["-32"]
+
+        if data_model == "LP64":
+            cmd = cmd + ["-64"]
```

Comment:
Many tool-info modules check whether the option is already present, and if yes, do not add it again. This keeps old benchmark definitions working. Not crucial, but maybe nice to have?

There is also a utility function in `sv_benchmarks_util`, which you could use, for example like this:
```python
        data_model_param = get_data_model_from_task(
            task, {ILP32: "-32", LP64: "-64"}
        )

        if data_model_param and data_model_param not in options:
            cmd += data_model_param
```

Reply:
Thanks! Sounds good.